client/src/components/lobby/Lobby.js

```diff
@@ -88,15 +93,21 @@ const Lobby: React$AbstractComponent<{}, LobbyHandleT> =
         socket.off("user-is-not-ready");
       };
     });
-
+    useEffect(() => {
+      const playable = userList.reduce(
+        (aux: bool, user: UserInLobbyT) => aux && 'writtenCharacter' in user,
+        true,
+      );
+      setPlayable(playable);
+    }, [userList, setPlayable]);
     return (
       <div>
         <button onClick={handleLeaveRoom}>
           Leave
         </button>
         <ReadyForm />
         <UserList users={userList} />
-        <PlayButton />
+        {room?.owner === user?.username && <PlayButton disabled={!isPlayable}/>}
```

Comment:
Nit: If there's neither room nor user, the expression will be true.
This works for now, because there's no way you end up rendering this without having a room.
However, once the project grows, a bug might escape and will end up showing here.
Just to keep in mind.

Reply:
Yeah, I was thinking that when a component uses some context (user, game, or room), add an early return if they are null, to avoid rendering stuff that shouldn't be rendered and avoid making calls to the server. I'll add an issue and work on it later